src/internal/wasi/io/v0.2.0/streams/streams.wit.go

```diff
@@ -54,6 +64,16 @@ func (self *StreamError) Closed() bool {
 	return self.Tag() == 1
 }
 
+var stringsStreamError = [2]string{
+	"last-operation-failed",
+	"closed",
+}
+
+// String implements [fmt.Stringer], returning the variant case name of v.
+func (v StreamError) String() string {
+	return stringsStreamError[v.Tag()]
```

Comment:
Again a comment for the generator here: should we be handling "unknown" types (`Tag() >= 2`) or just assume that will "never happen" and a crash regardless?

Reply:
Memorializing conversation from IRL this past week in San Francisco:

"Invalid variant cases are a bug, and a panic/crash is the intended behavior."